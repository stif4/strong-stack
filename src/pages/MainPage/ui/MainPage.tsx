import React from 'react';
import { useTranslation } from 'react-i18next';
import { BugButton } from 'app/providers/errorBoundary';
import { Counter } from 'entities/Counter';

const MainPage = () => {
    const { t } = useTranslation();

    return (
        <div>
            <BugButton />
            <Counter />
            <div>{t('Главная страница')}</div>
        </div>
    );
};

export default MainPage;
