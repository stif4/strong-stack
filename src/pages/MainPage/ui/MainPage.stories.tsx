import React from 'react';
import { Meta, StoryFn } from '@storybook/react';

import MainPage from './MainPage';

export default {
    title: 'pages/MainPage',
    component: MainPage,
} satisfies Meta<typeof MainPage>;

const Template: StoryFn<typeof MainPage> = (args) => <MainPage {...args} />;

export const Normal = Template.bind({});
Normal.args = {};
