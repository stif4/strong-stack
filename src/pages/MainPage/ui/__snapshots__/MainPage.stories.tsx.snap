// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/MainPage Normal smoke-test 1`] = `
<div class="app dark">
  <div>
    <button class="src-shared-ui-Button-Button-module__button undefined">
      throw error
    </button>
    <div>
      Главная страница
    </div>
  </div>
</div>
`;
